src/pages/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styles from './HomePage.scss';
```

Comment:
Try to come up with a unified order of importing files. Say, Thrid aprty modules come first. Then you components. Then assets. Then styles. 

You can come up with yours, but try to keep it celan and consistent